ops/travis/should-deploy.sh

```diff
@@ -11,8 +11,9 @@ update_build_info() {
     commit_time="$(git show -s --format=%ci HEAD)"
     deploy_time="$(date)"
     travis_job="$TRAVIS_BUILD_ID"
+    endpoints_sha=$(sha1sum tbaMobilev9openapi.json | awk '{ echo $1 }')
 
-    http $SET_BUILD_STATUS X-TBA-Auth-Key:$APIv3_KEY current_commit="$current_commit" commit_time="$commit_time" deploy_time="$deploy_time" travis_job="$travis_job"
+    http $SET_BUILD_STATUS X-TBA-Auth-Key:$APIv3_KEY current_commit="$current_commit" commit_time="$commit_time" deploy_time="$deploy_time" travis_job="$travis_job" endpoints_sha="$endpoints_sha"
```

Comment:
You're still going to need to have the server read this value here: https://github.com/the-blue-alliance/the-blue-alliance/blob/master/controllers/apiv3/api_admin_controller.py#L32